src/controllers/appointments.js

```diff
@@ -0,0 +1,11 @@
+const { getByDateService } = require('../services/appointments');
+
+const getByDate = async (req, res) => {
```

Comment:
Lets use collection name in controllers naming. For instance: getAppointmentsByDate